test/runtests.jl

```diff
@@ -1,7 +1,9 @@
+using Pkg
+
 Pkg.add("JuMP")
-Pkg.add("GLPKMathProgInterface")
+Pkg.add("GLPK")
```

Comment:
Remove `add` from `runtests.jl`. Create `test/REQUIRE` and add `JuMP` and `GLPK` there.